### PR TITLE
Frontier - add MNTL AssetMantle

### PR DESCRIPTION
**Pull request by Galadrin**

### CODE REVIEW

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -1608,6 +1608,35 @@
...
+            "umntl"
+          ]
+        },
+        {
+          "denom": "mntl",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/CBA34207E969623D95D057D9B11B0C8B32B89A71F170577D982FDDE623813FFC",
+      "name": "Mantle",
```
Comment (assetMantleAdmin) [inline osmosis-1/osmosis-frontier.assetlist.json:1628]: Can you please change this to `AssetMantle`?

**Comment by JeremyParish69** [comment]
LGTM. Pending liquidity

**Comment by JeremyParish69** [comment]
Pending valid coingeckoID

**Comment by JeremyParish69** [comment]
Pending valid coingeckoID

**Comment by IbrarMakaveli** [comment]
CoingeckoID it's not mandatory, thanks @JeremyParish69 , Merged 👍

**Pull request merged**